Source/ConversationV1/Models/MessageRequest.swift

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+//Struct for Messages
+public struct MessageRequest: JSONEncodable, JSONDecodable {
+    
+    var input:   [String: JSON]!
```

Comment:
These properties should probably be `private` since they are not accessed from outside of this class. (Only the `init(message:tags:context:)` and `toJSON()` functions need to be accessible from outside of this class.)